crates/starknet-server/src/api/http/endpoints/blocks.rs

```diff
@@ -1,11 +1,39 @@
-use axum::Json;
+use axum::{Extension, Json};
 
 use crate::api::http::error::HttpApiError;
-use crate::api::http::models::{AbortedBlocks, AbortingBlocks, CreatedBlock};
-use crate::api::http::HttpApiResult;
+use crate::api::http::models::{AbortedBlocks, AbortingBlocks, BlockInfo};
+use crate::api::http::{HttpApiHandler, HttpApiResult};
 
-pub(crate) async fn create_block() -> HttpApiResult<Json<CreatedBlock>> {
-    Err(HttpApiError::GeneralError)
+pub(crate) async fn create_block(
+    Extension(state): Extension<HttpApiHandler>,
+) -> HttpApiResult<Json<BlockInfo>> {
+    let mut starknet = state.api.starknet.write().await;
+    starknet.update_pending_block_timestamp();
+    let _ = starknet.create_block().map_err(|_| HttpApiError::CreateEmptyBlockError);
+    let last_block = starknet.get_latest_block();
+
+    match last_block {
+        Ok(block) => Ok(Json(BlockInfo {
+            block_hash: block.block_hash(),
+            block_timestamp: block.timestamp().0,
+        })),
+        Err(_err) => Err(HttpApiError::SetTimeError),
```

Comment:
Could this error be changed to something related to blocks, I guess its mistakenly transferred from time.rs

Reply:
Fixed in 0e0b7e706a639484868776952f6adba0df84ce07

Reply:
Looking at `set_time` and `create_block` both methods do the same thing

Reply:
but set set_time has input with timestamp and create_block doesn't